.github/workflows/macos.yml

```diff
@@ -205,24 +199,34 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@8d9ed9ac5c53483de85588cdf95a591a75ab9f55 # v5.5.0
       with:
-        python-version: ${{ matrix.python-version }}
-        cache: 'pip'
+        python-version: "3.13"
 
-    - name: Build and Install SciPy
+    - name: Make gfortran-13 on runner image usable
       run: |
-        sudo xcode-select -s /Applications/Xcode_16.app
+        # Ensure we use gfortran-13 and that its runtime is on the library search path
```

Comment:
on the macOS-15 runners there's gfortran-12/13/14, I can't remember the implications of using too old/too modern.

Reply:
Me neither, I decided to just not touch it since 13 is new enough.